apps/mobile/src/locales/en/messages.po

```diff
@@ -28,7 +28,7 @@ msgstr "{currency}"
 
 #: src/features/approver/components/fees/bitcoin-fee-option.tsx:40
 msgid "fees-sheet.fee-rate-caption"
-msgstr "{feeRate} sats/vB · {quoteFee}"
+msgstr "{feeRate} sats/vB · {fiatFee}"
```

Comment:
I refactored most instances of ‘fiat’ to be ‘quite’ so I think we could need to change this to ‘quoteFee’?